client/packages/host/src/Help/FeedbackForm.tsx

```diff
@@ -36,8 +53,10 @@ export const FeedbackForm = () => {
         Input={
           <BasicTextInput
             value={draft.replyEmail}
-            onChange={e => updateDraft({ replyEmail: e.target.value })}
+            onChange={handleChange}
             fullWidth
+            helperText={emailError}
+            error={!isValidEmail && draft.replyEmail !== ''}
```

Comment:
```suggestion
            error={!!emailError && draft.replyEmail !== ''}
```

and can then get rid of `isValidEmail` const.

Shouldn't need the extra isValidEmail check, you've already done that in the hook!

--- Actually, wondering if we should take off the `&& draft.replyEmail !== ''` part... I don't think it makes sense for the red border to sometimes go away, but the helper text to stay. Either not-empty check for both fields, or neither 🙏

Reply:
Fixed in f0151a536631c474c6d0ff633a26b9b26073d9db 😃